README.md

```diff
@@ -10,16 +10,18 @@
 
 **Tuono is the fastest React framework out there**
 
-Tuono (Italian word for "thunder", pronounced /2 Oh No/). 
+Tuono (Italian word for "thunder", pronounced /2 Oh No/).
 Why Tuono? Just a badass name.
 
-Some of its features are: 
+If you have experience with [NextJS](https://nextjs.org/) you will feel home here.
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The official spelling of this programming framework is “Next.js”. (NODE_JS[3])
Suggestions: `Next.js`
Rule: https://community.languagetool.org/rule/show/NODE_JS?lang=en-US&subId=3
Category: COMPOUNDING
<!-- __reviewdog__:ChAyOTc0ZjNkN2U5MWRjYjllEgxMYW5ndWFnZVRvb2w= -->